mne/viz/_brain/_brain.py

```diff
@@ -789,9 +793,111 @@ def update_lut(self, fmin=None, fmid=None, fmax=None, transparent=True):
 
         return self._data['ctable']
 
-    @property
-    def overlays(self):
-        return self._overlays
+    def set_data_smoothing(self, n_steps):
+        """Set the number of smoothing steps.
+
+        Parameters
+        ----------
+        n_steps : int
+            Number of smoothing steps
+        """
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", category=FutureWarning)
```

Comment:
What warning is this catching? It's best just to catch the one offending line if possible, and also add a comment saying what is being caught so that we limit the scope and hopefully eventually remove it